docs/markup-expr.md

```diff
@@ -3,5 +3,85 @@ id: markup-expr
 title: Expressions
 ---
 
-**TBD**
+Ergo Logic
```

Comment:
This feels like the wrong level of description (Indeed, Ergo is used for contract execution, but here it's about dynamic text in the contract).